packages/@webex/plugin-meetings/src/media/properties.ts

```diff
@@ -7,7 +7,7 @@ import LoggerProxy from '../common/logs/logger-proxy';
  * @class MediaProperties
  */
 export default class MediaProperties {
-  audioTrack: any;
+  audioTrack: LocalTrack | null;
```

Comment:
same here

Reply:
fixed - the base class `LocalTrack` is not used anywhere anymore